frontend/packages/akr/src/components/publicTranslator/listing/PublicTranslatorListingRow.tsx

```diff
@@ -80,9 +80,20 @@ export const PublicTranslatorListingRow = ({
   };
 
   const getLanguagePairs = useCallback(() => {
-    return languagePairs.map(({ from, to }, k) => {
-      const className =
-        fromLang === from && toLang === to ? 'padding-unset bold' : '';
+    const foundLangPairs = languagePairs.filter(({ from, to }) => {
+      return from === fromLang && to === toLang;
+    });
+    const otherLangPairs = languagePairs.filter(({ from, to }) => {
+      return !(from === fromLang && to === toLang);
+    });
+
+    let className = '';
+
+    return [...foundLangPairs, ...otherLangPairs].map(({ from, to }, k) => {
+      if (foundLangPairs.length) {
+        className =
+          k < foundLangPairs.length ? 'padding-unset bold' : 'lighter';
+      }
```

Comment:
Tämä nyt sorttaa kielet uudestaan siten että ensimmäisenä näkyy matchaava kielipari, ja muut kieliparit ovat alkuperäisessä (sekavassa) järjestyksessä. Koodi pysyisi yksinkertaisempana jos matchaavaa kieliparia ei nosteta ensimmäiseksi.

Reply:
Ymmärsin että saavutettavuuden näkökulmasta halutaan juuri hakuehdoilla löydetty kielipari aina ensimmäisenä, mutta muut voisi toki järjestää aakkosjärjestykseen esim. juuri "from" ja "to" puolesta.

Reply:
Tästä vois viedä ehkä sellaisen version untuvalle ja Otto vois siellä pyöritellä ja kommentoida tätä?

Reply:
Joo, viedään Untuvalle testattavaksi.